src/service.rs

```diff
@@ -36,21 +36,13 @@ use crate::rpc::{
 pub(crate) mod evm;
 use evm::EthConfiguration;
 
-#[cfg(not(feature = "runtime-benchmarks"))]
-type HostFunctions = sp_io::SubstrateHostFunctions;
-
-#[cfg(feature = "runtime-benchmarks")]
-type HostFunctions = (
-	sp_io::SubstrateHostFunctions,
-	frame_benchmarking::benchmarking::HostFunctions,
-);
-
-type FullClient<RuntimeApi> = TFullClient<Block, RuntimeApi, WasmExecutor<HostFunctions>>;
+type FullClient<RuntimeApi, Executor> =
+	TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>;
```

Comment:
**CR**

Why the change to `NativeElseWasmExecutor`?

Reply:
Keeping it for now as I am tired of changing these things back. @wischli would appreciate a post mortem clarification.

Reply:
Quick reply before Monday: This is taken from the cumulus parachain client.

Reply:
Fine for me.